app/uk/gov/hmrc/gform/views/form/addToList.scala.html

```diff
@@ -124,7 +129,7 @@
       @{hiddenFieldsSnippets}
 
         <div class="govuk-button-group">
-           @govukButton(Button(content = Text(buttonLabel), inputType=Some("submit"), value=Some("SaveAndContinue"), name=Some("save")))
+           @govukButton(Button(content = Text(button), inputType=Some("submit"), value=Some(button), name=Some("save")))
```

Comment:
I dont think the button value is used in the backend. If not, can we remove it please?

Reply:
Agreed. Removed.